packages/components/src/VirtualizedList/RowLarge/RowLarge.component.js

```diff
@@ -12,55 +20,76 @@ const { LARGE } = listTypes;
 /**
  * Row renderer that displays a Large item
  */
-function RowLarge({ className, index, parent, style }) {
-	const { titleField, selectionField, otherFields } = extractSpecialFields(parent);
+class RowLarge extends React.Component {
+	render() {
+		const { className, index, onKeyDown, parent, style } = this.props;
+		const { titleField, selectionField, otherFields } = extractSpecialFields(parent);
 
-	const parentId = getId(parent);
-	const id = parentId && `${parentId}-${index}`;
-	const titleCell = titleField && renderCell(index, parent, titleField, LARGE);
-	const selectionCell = selectionField && renderCell(index, parent, selectionField);
-	const rowData = getRowData(parent, index);
-	const otherCellsListItems = otherFields.map((field, fieldIndex) => {
-		const cellContent = renderCell(index, parent, field);
-		const tooltip = typeof cellContent === 'string' ? cellContent : null;
-		const label = getLabel(field);
-		return (
-			<li key={fieldIndex}>
-				{label && <span className={theme['field-label']}>{label}: </span>}
-				<span className={theme['field-value']} title={tooltip}>
-					{cellContent}
-				</span>
-			</li>
-		);
-	});
+		const parentId = getId(parent);
+		const id = parentId && `${parentId}-${index}`;
+		const titleCell = titleField && renderCell(index, parent, titleField, LARGE);
+		const selectionCell = selectionField && renderCell(index, parent, selectionField);
+		const rowData = getRowData(parent, index);
 
-	let onRowClick;
-	let onRowDoubleClick;
-	if (parent.props.onRowClick) {
-		onRowClick = event => parent.props.onRowClick({ event, rowData });
-	}
-	if (parent.props.onRowDoubleClick) {
-		onRowDoubleClick = event => parent.props.onRowDoubleClick({ event, rowData });
-	}
+		let onRowClick;
+		let onRowDoubleClick;
+		if (parent.props.onRowClick) {
+			onRowClick = event => parent.props.onRowClick({ event, rowData });
+		}
+		if (parent.props.onRowDoubleClick) {
+			onRowDoubleClick = event => parent.props.onRowDoubleClick({ event, rowData });
+		}
 
-	return (
-		<div
-			className={classNames('tc-list-item', rowThemes, rowData.className)}
-			role="button"
-			onClick={onRowClick}
-			onDoubleClick={onRowDoubleClick}
-			style={style}
-			key={id}
-		>
-			<div className={`tc-list-large-row ${theme['inner-box']} ${className}`} id={id}>
-				<div className={theme.header}>
-					{titleCell}
-					{selectionCell}
+		return (
+			// eslint-disable-next-line jsx-a11y/no-static-element-interactions
+			<div
+				className={classNames(
+					'tc-list-item',
+					'tc-list-large-row',
+					rowThemes,
+					rowData.className,
+					className,
+				)}
+				id={id}
+				onClick={onRowClick}
+				onDoubleClick={onRowDoubleClick}
+				onKeyDown={e => onKeyDown(e, this.ref)}
+				style={style}
+				ref={ref => {
+					this.ref = ref;
+				}}
+				role="listitem"
+				tabIndex="0"
```

Comment:
this field should be dynamic i think since this component is mapped from a value, anyway since tabIndex have a global incidence on the page, im not sure it will play well with other tabIndex definition outside of this list.

Reply:
I don't think so. `tabindex` 0 just add the elements in the natural html order flow. If used with value > 0, yes it would have a global incidence and this kind of thing if highly discouraged.
Here it's like I used a button element instead.